style.css

```diff
@@ -399,31 +406,166 @@ form > * {
   color: #ebebff;
 }
 
-form > input:focus {
-  border: 2px solid #9490f5;
+.desk {
+  display: none;
+}
+
+.elementRight-1 {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  .toolbar {
+    max-height: 72px;
+    max-width: 100%;
+    justify-content: space-evenly;
+  }
+
+  .logo {
+    line-height: 20px;
+  }
+
+  .nav > .button-nav {
+    display: none;
+  }
+
+  nav > ul {
+    display: flex;
+    gap: 8px;
+  }
+
+  nav > ul > .nav-link1 {
+    padding-bottom: 8px;
+  }
+
+  .sub-container-1 {
+    background-image: url(icons&imgs/twoCirclesHeader.png);
+  }
+
+  .elementRight-1 {
+    display: flex;
+  }
+
+  .headline {
```

Comment:
- [ ] Your headline still looks great 👏. You really have done much so far. What I would suggest to improve on is your heading in this section, which is your name. It looks like it needs more styling in order to resemble the design you chose. Look at the comparison below 👇 
**Your design**
![image](https://user-images.githubusercontent.com/90222110/177840648-111d89fb-db4b-4f2e-b1f4-a13d3e72cc69.png)
**Figma design**
![image](https://user-images.githubusercontent.com/90222110/177840810-0cab0a7d-6058-4551-a81a-d41c013180d7.png)
Please try to more styling to your heading(which is your name) in order to resemble the Figma design. This also makes your name **more in focus** so that when someone lands on your page, he/she will see your name first 😃 .